# Defines the discretized states of Hyq with joint angles

std_msgs/Float64 dt    # the time [s] between two states
HyqState[] states      # trajectory of states